lib/lrama/grammar/stdlib.y

```diff
@@ -0,0 +1,80 @@
+/**********************************************************************
+
+  stdlib.y
+
+  This is lrama's standard library. It provides a number of
+  parameterizing rule definitions, such as options and lists,
+  that should be useful in a number of situations.
+
+**********************************************************************/
+
+/*
+ * program: option(number)
+ *
+ * =>
+ *
+ * program: option_number
+ * option_number: ε
+ * option_number: number
+ */
+%rule option(X): /* empty */
+               | X
+               ;
+
+/*
+ * program: list(number)
+ *
+ * =>
+ *
+ * program: list_number
+ * list_number: ε
+ * list_number: number list_number
+ */
+%rule list(X): /* empty */
+             | X list(X)
+             ;
+
+/*
+ * program: nonempty_list(number)
+ *
+ * =>
+ *
+ * program: nonempty_list_number
+ * nonempty_list_number: number
+ * nonempty_list_number: number nonempty_list_number
+ */
+%rule nonempty_list(X): X
+                      | X nonempty_list(X)
```

Comment:
The order of RHS tokens are changed, is this intentional?

Reply:
Ditto. https://gitlab.inria.fr/fpottier/menhir/-/blob/20230608/src/standard.mly?ref_type=tags#L219